src/docs/configuration.md.templar

```diff
@@ -299,6 +299,31 @@ The following attributes are a part of the Patching DSL <kbd>patchPluginXml { ..
 
+### Plugin Verifier DSL
+[IntelliJ Plugin Verifier](https://github.com/JetBrains/intellij-plugin-verifier) integration task allows to check the binary compatibility of the built plugin against the specified [IntelliJ IDE builds](https://jetbrains.org/intellij/sdk/docs/reference_guide/api_changes_list.html).
+
+Plugin Verifier DSL `pluginVerifier { ... }` allows to define the list of IDEs used for the verification, as well as explicit tool version and any of the available [options](https://github.com/JetBrains/intellij-plugin-verifier#common-options) by proxifying them to the Verifier CLI.
+
+> **TIP** For more details, examples or issues reporting, go to the [IntelliJ Plugin Verifier](https://github.com/JetBrains/intellij-plugin-verifier) repository.
+
+| **Attributes**                                                                                                                                                 | **Default Value**                                      |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| <kbd>ides</kbd> - Array of IDEs to check, in `intellij.version` format, like: `IC-2020.2`; if not set, uses `intellij`. Should be set for the best experience. | <kbd>["${intellij.type}-${intellij.version}"]</kbd>    |
+| <kbd>verifierVersion</kbd> - IntelliJ Plugin Verifier version, by default uses the latest available. It's recommended to use always the latest version.        | <kbd>latest</kbd>                                      |
+| <kbd>failureLevel</kbd> - Defines the verification level at which task should fail. Can be set as `FailureLevel` enum or `EnumSet<FailureLevel>`.              | <kbd>FailureLevel.INVALID_PLUGIN</kbd>                 |
+| <kbd>verificationReportsDir</kbd> - The path to directory where verification reports will be saved.                                                            | <kbd>${project.buildDir}/reports/pluginsVerifier</kbd> |
+| <kbd>jbrVersion</kbd> - JBR version used by the Verifier.                                                                                                      | none                                                   |
+| <kbd>runtimeDir</kbd> - The path to directory containing Java runtime jar, overrides JBR.                                                                      | none                                                   |
+| <kbd>externalPrefixes</kbd> - The prefixes of classes from the external libraries.                                                                             | none                                                   |
+| <kbd>pluginsToCheckAllBuilds</kbd> - The plugin IDs to check ALL compatible plugin builds.                                                                     | none                                                   |
```

Comment:
I don't understand that. What should I set here? Considering it has a String type

Reply:
I've copy&pasted description from the Verifier docs - it's not so clear.
Anyway, it's used for building the verification matrix: multiple plugins x IDEs; since we're going to test only one current plugin against n IDEs, it's pointless to expose that property.
Removed.